### PR TITLE
[OVERFLOW-22] Implement Comments Form Component

### PR DESCRIPTION
**Pull request by kenorenzsun**
## Backlog Link
[SUN_OVERFLOW-22](https://framgiaph.backlog.com/view/SUN_OVERFLOW-22)
## Commands
None
## Pre-conditions
None
## Expected Output
Shows comment form component
## Notes
None
## Screenshots
image

### CODE REVIEW

**Review by nicoleamber — APPROVED** [review]

File: frontend/components/CommentForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        formState: { errors },
+    } = useForm<FormValues>({
+        mode: 'onSubmit',
+        reValidateMode: 'onSubmit',
+        resolver: yupResolver(CommentFormSchema),
+    })
+
+    const onSubmit = (data: FormValues) => {
+        // id - you can use this to condition between update and create
+        if (isObjectEmpty(errors)) {
```
Comment (IanUrriza10) [inline frontend/components/CommentForm/index.tsx]: This isn't needed anymore according to gift as it interferes when there's an error in the initial send.

File: frontend/components/CommentForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            //Replace with graphql post request
+            console.log(data)
+        }
+    }
+
+    useEffect(() => {
+        register('comment', {})
+        setValue('comment', '')
+    }, [register])
```
Comment (IanUrriza10) [inline frontend/components/CommentForm/index.tsx]: I don't think this is needed anymore as you have already declared register in the text area.

**Comment by ipilikent** [comment]
## Issues 
- Raise the comment box a bit
- Make the add comment slightly higher
- Make the Submit Comment bar a little higher
- The color of the submit answer is the same as its border

## Screenshot/GIF
<details>

22 Comment

</details>

**Comment by ipilikent** [comment]
## Approved PR
- All displayed the same as the Figma design
- Comment Box, Submit Comment Button is displayed
- Can clicked the Add Comment

## Proof
<details>

22 -Markup Comment Form Component

22 -Markup Comment Form Component

</details>

Test Case: https://docs.google.com/spreadsheets/d/114GrEn-LQQ8saIvjz9zqenuk05qmO4dN/edit#gid=1031094368 
-----
- Total Test time: 0.5
- Total Comments: 4
- Total Iterations: 1

**Pull request merged**